client/package-lock.json

```diff
@@ -4974,12 +4974,13 @@
       "dev": true
     },
     "node_modules/@types/node": {
-      "version": "22.7.4",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-22.7.4.tgz",
-      "integrity": "sha512-y+NPi1rFzDs1NdQHHToqeiX2TIS79SWEAw9GYhkkx8bD0ChpfqC+n2j5OXOCpzfojBEBt6DnEnnG9MY0zk1XLg==",
+      "version": "22.9.0",
```

Comment:
In general, best to keep package updates in a separate PR unless relevant to that PR

Reply:
I'm not sure how this happened. I'll fix this.